ipywidgets/static/notebook/js/manager.js

```diff
@@ -4,17 +4,16 @@
 define([
     "underscore",
     "backbone",
-    "./utils",
     "services/kernels/comm",
-    "./manager-base"
-], function (_, Backbone, utils, comm, managerBase) {
```

Comment:
all the exports of utils are exported in `jupyter-js-widgets`. Here we are out of `jupyter-js-widgets`.